tests/e2e/test_api_xai_sanity.py

```diff
@@ -0,0 +1,217 @@
+            task = ClassificationTrainTask(task_environment=task_environment)
+            train_parameters = TrainParameters()
+            output_model = ModelEntity(
+                dataset,
+                task_environment.get_model_configuration(),
+            )
+            task.train(dataset, output_model, train_parameters)
+            save_model_data(output_model, temp_dir)
+
+            for processed_saliency_maps, only_predicted in [[True, False], [False, True]]:
+                task_environment, dataset = self.init_environment(
+                    hyper_parameters, model_template, multilabel, hierarchical, 20
+                )
+
+                # Infer torch model
+                task = ClassificationInferenceTask(task_environment=task_environment)
+                inference_parameters = InferenceParameters(
+                    is_evaluation=False,
+                    process_saliency_maps=processed_saliency_maps,
+                    explain_predicted_classes=only_predicted,
+                )
+                predicted_dataset = task.infer(dataset.with_empty_annotations(), inference_parameters)
+
+                # Check saliency maps torch task
+                task_labels = output_model.configuration.get_label_schema().get_labels(include_empty=False)
+                saliency_maps_check(
+                    predicted_dataset,
+                    task_labels,
+                    self.ref_raw_saliency_shapes[model_template.name],
+                    processed_saliency_maps=processed_saliency_maps,
+                    only_predicted=only_predicted,
+                )
+
+                # Save OV IR model
+                task._model_ckpt = osp.join(temp_dir, "weights.pth")
+                exported_model = ModelEntity(None, task_environment.get_model_configuration())
+                task.export(ExportType.OPENVINO, exported_model, dump_features=True)
+                os.makedirs(temp_dir, exist_ok=True)
+                save_model_data(exported_model, temp_dir)
+
+                # Infer OV IR model
+                load_weights_ov = osp.join(temp_dir, "openvino.xml")
+                task_environment.model = read_model(task_environment.get_model_configuration(), load_weights_ov, None)
+                task = ClassificationOpenVINOTask(task_environment=task_environment)
+                _, dataset = self.init_environment(hyper_parameters, model_template, multilabel, hierarchical, 20)
+                predicted_dataset_ov = task.infer(dataset.with_empty_annotations(), inference_parameters)
+
+                # Check saliency maps OV task
+                saliency_maps_check(
+                    predicted_dataset_ov,
+                    task_labels,
+                    self.ref_raw_saliency_shapes[model_template.name],
+                    processed_saliency_maps=processed_saliency_maps,
+                    only_predicted=only_predicted,
+                )
+
+
+# class TestOVDetXAIAPI(DetectionTaskAPIBase):
```

Comment:
Can we remove these lines?

Reply:
It is better to mark it with `pytest.mark.skip`, but there are invalid imports so I just commented it out. I did not remove it because I will refactor it in the near future and enable back (I cannot do it fast because det task was refactored). Can we keep it like this temporary?